cmd/webrpc-gen/main.go

```diff
@@ -16,19 +17,38 @@ var flags = flag.NewFlagSet("webrpc-gen", flag.ExitOnError)
 func main() {
 	versionFlag := flags.Bool("version", false, "print webrpc version and exit")
 	schemaFlag := flags.String("schema", "", "webrpc schema file (required)")
-	pkgFlag := flags.String("pkg", "proto", "generated package name for target language, default: proto")
+	targetFlag := flags.String("target", "", fmt.Sprintf("target generator for webrpc library generation (required), ie. github.com/webrpc/gen-golang@v0.6.0"))
```

Comment:
I think just say.... `" .. ie. golang@v0.6.0"`     .. this is now supported and recommended as of v0.6.2